applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -546,7 +546,7 @@ static void on_device_found(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
 			bt_foreach_bond(BT_ID_DEFAULT, bond_connect, (void *)addr);
 		}
 		return;
-	} else if (type == BT_GAP_ADV_TYPE_ADV_IND) {
+	} else if (type == BT_GAP_ADV_TYPE_ADV_IND || type == BT_GAP_ADV_TYPE_EXT_ADV) {
```

Comment:
So we need to include BT_GAP_ADV_TYPE_ADV_IND here since this is used if the gateway is bonded with headset(s)?

Reply:
No, that's BT_GAP_ADV_TYPE_ADV_DIRECT_IND. The BT_GAP_ADV_TYPE_ADV_IND  is the old type of advertising, I believe, so it seems we will support both legacy and ext adv going forward

Reply:
I don't want to remove the ADV_IND in the current stage.
Since there user might want to connect to other devices with using legacy ADV (like what we seen in UPF, there still some devices using legacy ADV instead of ADV-EXT).